### ISSUE TITLE
Typing `cd` in console breaks it

### ISSUE THREAD

**Issue by jdel** [coderaiser/cloudcmd#111]
** coderaiser/cloudcmd:6.8.0-alpine ** 

When using the web console, typing `cd` in it will break cloudcmd with:

```
assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: command could not be empty!
    at Object.isChangeVolume (/usr/src/app/node_modules/win32/legacy/win.js:116:5)
    at Spawnify.onCD [as _onCD] (/usr/src/app/node_modules/spawnify/legacy/spawnify.js:246:31)
    at Spawnify.onMessage [as _onMessage] (/usr/src/app/node_modules/spawnify/legacy/spawnify.js:80:14)
    at /usr/src/app/node_modules/spawnify/legacy/spawnify.js:49:16
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

The console will display `console: disconnected` and the container needs to be restarted for the console to work again.

I'm not too sure why there is a `win32` reference here. I'm running the container on a Linux kernel...

**Comment by coderaiser** [comment]
Thank you, fixed with https://github.com/coderaiser/spawnify/commit/38b4d58bad2adec6746e6b07edaf31461cc066f5 in [spawnify v3.0.1](https://github.com/coderaiser/spawnify/releases/tag/v3.0.1).

**Comment by coderaiser** [comment]
Landed in [v6.8.1](https://github.com/coderaiser/cloudcmd/releases/tag/v6.8.1), `coderaiser/cloudcmd:6.8.1` and `coderaiser/cloudcmd:6.8.1-alpine`
